kafka-client/pom.xml

```diff
@@ -29,11 +29,12 @@
             <groupId>org.jenkins-ci.main</groupId>
             <artifactId>remoting</artifactId>
             <version>${remoting.version}</version>
+            <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>com.fasterxml.jackson.core</groupId>
-            <artifactId>jackson-databind</artifactId>
-            <version>2.9.5</version>
+            <groupId>org.jenkins-ci.plugins</groupId>
```

Comment:
It won't work. Plugins should be included only into Plugins, but here you make Kafka Agent binary to depend on the Plugin.

Reply:
@oleg-nenashev if not then how can I use `jackson2-api` as you suggested above